decam/makeCrosstalkDecam.py

```diff
@@ -6,9 +6,15 @@
 import os.path
 import sys
 
+import lsst.obs.decam
 import lsst.ip.isr as ipIsr
 from lsst.daf.base import PropertyList
-from lsst.obs.decam import DecamMapper
+from lsst.utils import getPackageDir
+
+
+def getCrosstalkDir():
+    """Get crosstalk directory in obs package for outputting data."""
+    return os.path.join(getPackageDir('obs_decam'), 'decam', 'crosstalk')
```

Comment:
I think it's highly likely that you are going to want people to specify an input directory or a glob expanding to all the files rather than expecting the crosstalk files to always be here.

Reply:
This is the destination, it just puts them in `decam/crosstalk` as before.